cmd/podman/varlink.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/ioprojectatomicpodman"
+	"github.com/projectatomic/libpod/pkg/varlinkapi"
+	"github.com/projectatomic/libpod/version"
+	"github.com/urfave/cli"
+	"github.com/varlink/go/varlink"
+)
+
+var (
+	varlinkDescription = `
+	podman varlink
+
+	run varlink interface
+`
+	varlinkFlags   = []cli.Flag{}
+	varlinkCommand = cli.Command{
+		Name:        "varlink",
+		Usage:       "Run varlink interface",
+		Description: varlinkDescription,
+		Flags:       varlinkFlags,
+		Action:      varlinkCmd,
+		ArgsUsage:   "VARLINK_URI",
+	}
+)
+
+func varlinkCmd(c *cli.Context) error {
+	args := c.Args()
+	if len(args) < 1 {
+		return errors.Errorf("you must provide a varlink URI")
+	}
+
+	var varlinkInterfaces = []*ioprojectatomicpodman.VarlinkInterface{varlinkapi.VarlinkLibpod}
+	// Register varlink service. The metadata can be retrieved with:
+	// $ varlink info [varlink address URI]
+	service, _ := varlink.NewService(
```

Comment:
Does this return an error? We probably should not ignore it if so

Reply:
nice catch, fixed